crawl-ref/source/cloud.cc

```diff
@@ -302,6 +302,11 @@ static const cloud_data clouds[] = {
       ETC_FIRE,                                   // colour
       { TILE_CLOUD_FLAME, CTVARY_RANDOM },        // tile
     },
+    // CLOUD_ELECTRICITY,
+    { "wisps of electric spark", nullptr,         // terse, verbose name
```

Comment:
"wisps of electric spark" isn't quite valid English. I'd recommend just 'sparks'.